compose/compose-file.md

```diff
@@ -256,9 +256,11 @@ updates.
 - `monitor`: Duration after each task update to monitor for failure `(ns|us|ms|s|m|h)` (default 0s).
 - `max_failure_ratio`: Failure rate to tolerate during an update.
 
+```
```

Comment:
I thought of that; arguably, the whitespace belongs, since this section of YAML does happen at a certain indentation level. What do you think?